### DISCUSSION TITLE
Gera um banco de dados com 5 tabelas e 5 linha e relacionado

### DISCUSSION THREAD

**Discussion by 19850201** [community/community#142014] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Gera um banco de dados com 5 tabelas e 5 linha e relacionado

**Comment by 19850201** [comment]
Gera um banco de dados com 5 tabelas e 5 linha e relacionado

**Comment by queenofcorgis** [comment]
Hi @19850201 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Please note:  You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
